src/vite-env.d.ts

```diff
@@ -1 +1,8 @@
 /// <reference types="vite/client" />
+
+declare module '*.svg' {
+  import React = require('react');
```

Comment:
```suggestion
// eslint-disable-next-line @typescript-eslint/no-require-imports
  import React = require('react');
```

eslint disable 라인이 있어야 오류가 안 뜨네요 우리 설정이 다른가